hi-ml-histopathology/src/histopathology/models/deepmil.py

```diff
@@ -352,20 +352,95 @@ def get_bag_label(labels: Tensor) -> Tensor:
         # SlidesDataModule attributes a single label to a bag of tiles already no need to do majority voting
         return labels
 
+    @staticmethod
+    def get_empty_lists(shape: int, n: int) -> List:
```

Comment:
this can be done in a single line `ll=[[None]*shape]*n` ? 
might need docs and only used once, not sure it's worth creating a function for it.

Reply:
`this can be done in a single line ll=[[None]*shape]*n` no it can't!
Initially I had used the same compact syntax ...and it costed me one day of debugging. If the lists are created as you suggested, they are pointer to the same value ... that means changing value in one of the lists will change also the other ones.  This is way I was ending up with top = bottom = left = right.

The alternative to have a function is to replicate the same syntax 4 times in the method below, that seemed less nice.. Il'll add docs